packages/protocol/contracts-0.8/common/UsingRegistry.sol

```diff
@@ -55,6 +55,8 @@ contract UsingRegistry is Ownable {
   bytes32 constant LOCKED_CELO_REGISTRY_ID = keccak256(abi.encodePacked("LockedCelo"));
   bytes32 constant CELO_UNRELEASED_TREASURE_REGISTRY_ID =
     keccak256(abi.encodePacked("CeloUnreleasedTreasure"));
+  bytes32 constant EPOCH_MANAGER_INITIALIZER_REGISTRY_ID =
```

Comment:
Initializer or Enabler ?

Reply:
the whoe renaming is done in a separate PR, so i'm keeping as is for now.